aten/src/ATen/native/TensorProperties.cpp

```diff
@@ -13,6 +14,21 @@ bool is_same_size(const Tensor& self, const Tensor& other) {
   return self.sizes().equals(other.sizes());
```

Comment:
Should I actually just update is_same_size() instead of adding a new native function that is essentially doing the same thing?

Reply:
Seems very reasonable to me.